client/src/main.ts

```diff
@@ -71,16 +92,20 @@ let document =
 //    to parse the source code. This gives us a list of `BlockKind` values
 //    (with `language` set to the right language)
 
+let index = 0;
 for (let cell of document) {
   var language = cell['language'];
   if (languagePlugins[language] == null)
     console.log("No language plugins for "+language);
   else 
   {
-    console.log("Language plugin for " + language + " is " + languagePlugins[language].language);
+    // console.log("Language plugin for " + language + " is " + languagePlugins[language].language);
+    let editorId = "editor_"+index;
+    $('#paper').append("<div id=\""+editorId+"\" style=\"height:100px;\"></div>")
```

Comment:
I have a slight preference for not using JQuery if we can do all we need relatively easily without it - there is nothing wrong with JQuery, but I think it's nice to keep the core dependencies smaller (although, language plugins will probably have their own dependencies and we cannot do much about that).

Right now, if we just need to find elements and append elements, it can be quite easily done using:

```
document.getElementById("whatever")
```

and also:

```
var div = document.createElement("div");
div.id = "whatever";
div.style.height = "100px"
document.getElementById("paper").appendChild(div)
```

This is a bit longer, but I think it's not too bad - but I guess if we end up doing more complicated UI things, then JQuery might be useful.